### ISSUE TITLE
Element with aria-hidden has no content in sequential focus navigation [6cfa84] - Passed example 4

### ISSUE THREAD

**Issue by giacomo-petri** [auto-wcag/auto-wcag#2232]
In [Element with aria-hidden has no content in sequential focus navigation [6cfa84]](https://www.w3.org/WAI/standards-guidelines/act/rules/6cfa84/), Failed Example 4 should be removed.

This rule maps to 4.1.2 Name, Role, Value, but this success criterion specifically applies to [user interface components](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html#dfn-user-interface-component), which are defined as:
> a part of the content that is perceived by users as a single control for a distinct function

The <p> element in Failed Example 4 does not fit this definition. To align with the mapping to SC 4.1.2, this example should be removed.

**Comment by carlosapaduarte** [comment]
Decision: 
4.1.2 becomes secondary requirement
1.3.1 becomes primary requirement
Review examples and assumptions
